bin/dm_parse_faces.py

```
@@ -0,0 +1,251 @@
+            trial_end = np.empty([len(taglist)], dtype=int)
+
+            for i, ind_trial_proc in enumerate(taglist):
+                if (i < (len(taglist)) - 1):
+                    trial_end[i] = taglist[i + 1][0]
+                elif (i == (len(taglist)) - 1):
+                    trial_end[i] = len(eprime) - 1
+
+                trial_start[i] = ind_trial_proc[0]
+
+            trial_blocks = [
+                eprime[s:e] for s, e in zip(trial_start, trial_end)
+            ]
+
+            entries = []
+            for b in trial_blocks:
+                entries.append({
+                    'onset':
+                    get_event_value(b, 'StimSlide.OnsetTime:'),
+                    'duration':
+                    get_event_value(b, 'StimSlide.OnsetToOnsetTime:'),
+                    'trial_type':
+                    'Shapes' if 'Shape' in str(b) else 'Faces',
+                    'response_time':
+                    get_event_value(b, 'StimSlide.RT:'),
+                    'accuracy':
+                    get_event_value(b, 'StimSlide.ACC:'),
+                    'correct_response':
+                    map_response(get_event_value(b, 'CorrectResponse:')),
+                    'participant_response':
+                    map_response(get_event_value(b, 'StimSlide.RESP:'))
+                })
+
+            data = pd.DataFrame.from_dict(entries)\
+                .astype({
+                    "onset": np.float,
+                    "duration": np.float,
+                    "response_time": np.float,
+                    "accuracy": np.float,
+                    "correct_response": np.float,
+                    "participant_response": np.float
+                })\
+                .astype({
+                    "correct_response": "Int64",
+                    "participant_response": "Int64",
+                    "accuracy": "Int64"
+                })
+
+            log_head, log_tail = os.path.split(eprimefile)
+
+            if not out_dir:
```

`    --output-dir OUT_DIR    Specify an alternate output directory                            [Default:                            $DATMAN_PROJECTSDIR/{study}/data/nii/{session}]`

Wouldn't `out_dir` never be empty since there's a default value? Please correct me if I am wrong.

---

`out_dir` would be empty if the `--output-dir` arg isn't specified. The default in the docstring is just so the user knows it'll be put into the configured archive folder, it won't actually set a value (the docstring parser will silently fail to make it a real value).